lib/matplotlib/backends/_backend_tk.py

```diff
@@ -515,9 +515,9 @@ def __init__(self, canvas, window, *, pack_toolbar=True):
         # lines high. Otherwise the canvas gets redrawn as the mouse hovers
         # over images because those use two-line messages which resize the
         # toolbar.
-        label = tk.Label(master=self,
+        self._label_filler = tk.Label(master=self,
                          text='\N{NO-BREAK SPACE}\n\N{NO-BREAK SPACE}')
```

Comment:
```suggestion
        self._label_filler = tk.Label(master=self,
                                      text='\N{NO-BREAK SPACE}\n\N{NO-BREAK SPACE}')
```

Reply:
I checked but testing is not passed because the text length is too long (84 >79).
I retry the another code, maybe it is same as the previous one.